draft-ietf-anima-brski-prm.md

```diff
@@ -336,84 +320,84 @@ Solution examples based on existing technology are provided with the focus on ex
-   |                         +--------------+---------+
-   |                                       ^
-   |                                       |  BRSKI-
-   |    BRSKI-PRM                          |   MASA
-   |          .............................|.........
-   V          .                            v        .
-+-------+     .  +-----------+       +-----------+  .
-|       |     .  |           |       |           |  .
-|Pledge |     .  | Registrar | BRSKI | Domain    |  .
-|       |     .  | Agent     |  EST  | Registrar |  .
-|       |<------>|           |<----->| (PKI RA)  |  .
-|       |     .  |     LDevID|       |           |  .
-|       |     .  +-----------+       +-----+-----+  .
-|IDevID |     .                            |        .
-|       |     .         +------------------+-----+  .
-+-------+     .         | Key Infrastructure     |  .
-              .         | (e.g., PKI Certificate |  .
-              .         |       Authority)       |  .
-              .         +------------------------+  .
-              .......................................
-                       "Domain" components
+                         +---------------------------+
+    +---- Drop Ship -----| Vendor Service            |
+    |                    +---------------+-----------+
+    |                    | M anufacturer |           |
+    |                    | A uthorized   | Ownership |
+    |                    | S igning      | Tracker   |
+    |                    | A uthority    |           |
+    |                    +---------------+-----------+
+    |                                         ^
+    |                                         | BRSKI-
+    |                                         | MASA
+    |          ...............................|.........
+    V          .                              v        .
++--------+     .  +------------+        +-----------+  .
+|        |     .  |            |        |           |  .
+| Pledge | BRSKI- | Registrar- | BRSKI- | Domain    |  .
+|        |  PRM   | Agent      |  EST   | Registrar |  .
+|        |<------>|            |<------>| (PKI RA)  |  .
+|        |     .  |     LDevID |        |           |  .
+|        |     .  +------------+        +-----+-----+  .
+| IDevID |     .                              |        .
+|        |     .           +------------------+-----+  .
++--------+     .           | Key Infrastructure     |  .
+               .           | (e.g., PKI Certificate |  .
+               .           |        Authority)      |  .
+               .           +------------------------+  .
+               .........................................
+                         "Domain" Components
 ~~~~
-{: #uc2figure title='BRSKI-PRM Architecture overview using registrar-agent' artwork-align="left"}
+{: #uc2figure title='BRSKI-PRM architecture overview using registrar-agent' artwork-align="left"}
 
 The following list describes the components in a (customer) site domain:
```

Comment:
In the figure, the pledge is still outside the domain. Why is it now in the site domain? (Yes, it wants in, but we have conflicting statements now.)

Reply:
The figure says the is pledge outside the domain, the sentence above the pledge bullet point says it is in the domain ("components in a (customer) site domain"). Neither the "The manufacturer provided components/services" are not inside the domain.

It should probably just say something like "Figure 1 shows the relations between the following main components:".

Reply:
Changed accordingly. Makes it clearer.